rpcs3/rpcs3qt/mainwindow.cpp

```diff
@@ -0,0 +1,233 @@
+#ifdef QT_UI
```

Comment:
Hmm, I think this is usless for current rpcs3qt b/c it's builds manually by calling qmake.